charts/Makefile

```diff
@@ -12,14 +12,19 @@ cert-manager:
 	kubectl create namespace cert-manager
 	helm repo add jetstack https://charts.jetstack.io
 	helm repo update
-	helm install cert-manager jetstack/cert-manager \
-	  --namespace cert-manager --version v1.1.0 --set installCRDs=true \
-	  --wait --timeout 120s
+	helm install cert-manager jetstack/cert-manager --version v1.1.0 \
+		--set installCRDs=true \
+		--namespace cert-manager \
+		--wait --timeout 120s
 
 .PHONY: vault
 vault:
 	@echo "Installing vault ..."
-	cd ../third_party/vault && $(MAKE) deploy deploy-wait
+	helm repo add hashicorp https://helm.releases.hashicorp.com
+	helm install vault hashicorp/vault --version 0.8.0 \
+                --set "global.openshift=false" \
```

Comment:
Why is it always false?

Reply:
this setup is on a local kind cluster, so its' false, in our instructions that we prepare later on we will explain setup for Openshift as well

Reply:
Eventually one should be able to install all charts on OpenShift as well (for evaluation) just like any other Kubernetes cluster.